### ISSUE TITLE
Si4732 frequency calibration

### ISSUE THREAD

**Issue by byteborg** [phdlee/uvk5cec#77]
Would there be a possibility to store oscillator calibration for the Si4732? I made the mod and my device seems to be about 1.49kHz too low at 14MHz which practically inhibits tuning to CW on a somewhat sane frequency grid. It'd be great to store oscillator calibration in firmware and be able to display and use the corrected frequency.

**Comment by Hoover99** [comment]
Yes, I've noticed the approx same frequency offset on my receiver.
